deployment/testing_environment/manifests/site.pp

```diff
@@ -0,0 +1,40 @@
+node default {
+    # needed for ruby2.4
+    exec { 'apt-get install software-properties-common':
+        provider => shell,
+        command  => 'apt-get update && apt-get install -y software-properties-common'
+    } ->
+    exec { 'apt-add-repository':
+        provider => shell,
+        command  => 'apt-add-repository -y ppa:brightbox/ruby-ng && apt-get update'
+    } ->
+    exec { 'apt-get update':
+        provider => shell,
+        command  => 'apt-get update'
+    }
+
+    # general packages
+    package { ['git', 'build-essential', 'vim']:
+        ensure => installed,
+    } ->
+    # python packages
+    package { ['python3', 'python3-dev', 'python3-pip', 'libxslt1-dev', 'zlib1g-dev', 'gettext', 'libpq-dev']:
+        ensure => installed,
+    }
+    ->
+    package { ['ruby2.4', 'ruby2.4-dev']:
```

Comment:
why do we need ruby in the vagrant image?

Reply:
Man kann es auch mal ohne probieren. Puppet ist halt ruby

Reply:
Alles klar, dann lassen wir es erstmal drinnen.